aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -186,7 +190,7 @@ Tensor& frac_(Tensor& self) { return unary_op_impl_(self, at::frac_out); }
 Tensor& floor_out(Tensor& result, const Tensor& self) {
   // Note: this is consistent with NumPy
   TORCH_CHECK(!self.is_complex(),
-    "floor is not supported for complex inputs");
+    "floor is not supported for complex inputs")
```

Comment:
I think this was by mistake? (removing `;`)

Reply:
You are right! Thanks for point that out 👍